src/engine/ruleset/environments/wazuh-environment.yml

```diff
@@ -18,6 +18,7 @@ decoders:
   - decoder/fim/0
   - decoder/fim-event/0
   - decoder/fim-scan/0
+  - decoder/redis/0
```

Comment:
Nitpick (Design): we could add the new decoders at the end with a comment, like so:

 # Redis decoders
  --decoder/redis/0

Reply:
I don't know, I saw that they are arranged alphabetically. what do you think about this? @cborla